std/zig.zig

```diff
@@ -2,6 +2,7 @@ const tokenizer = @import("zig/tokenizer.zig");
 pub const Token = tokenizer.Token;
 pub const Tokenizer = tokenizer.Tokenizer;
 pub const parse = @import("zig/parse.zig").parse;
+pub const parse2 = @import("zig/parse2.zig").parse;
```

Comment:
Is this intentional?

Reply:
Nope, artifact of the funky squish 'n' rebase. I'll fix.